tests/unit_tests/test_wave_06.py

```diff
@@ -142,8 +148,16 @@ def test_swap_best_by_category_reordered():
         my_priority="Clothing",
         their_priority="Decor"
     )
+    # Assert
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
 
-    raise Exception("Complete this test according to comments below.")
+    assert item_f in tai.inventory
```

Comment:
Good job again! You could still check every item just to be extra sure nothing was moved around unexpectedly.